src-java/ping-topology/ping-messaging/src/main/java/org/openkilda/messaging/info/flow/FlowPingResponseUtils.java

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2023 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.messaging.info.flow;
+
+import java.util.List;
+
+/**
+ * Utility class for {@link FlowPingResponse}.
+ */
+public final class FlowPingResponseUtils {
+
+    private FlowPingResponseUtils() {}
+
+    /**
+     * Check if ping was successful.
+     *
+     * @param error error message
+     * @param subFlows list of subflows
+     * @return true if ping was successful, false otherwise
```

Comment:
the statement is false. The ping could be successful, but this method still returns false. That leads to the question why the error parameter is here? Shouldn't it be in some other method? Otherwise it's like pingSuccess validates a string